ios/Runner/Base.lproj/Main.storyboard

```diff
@@ -1,8 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why do we need a storyboard here?

Reply:
I had problems with firebase version on IOS. I got the error when my code did not pass CI/CD progress in the previous commit. After check the error, looks like I need to update some packages (not sure) but I ran this line of code 
"pod repo update"
Then it automatically change that file